depth_image_proc/launch/convert_metric.launch.py

```diff
@@ -45,7 +45,7 @@ def generate_launch_description():
     return LaunchDescription([
         # install realsense from https://github.com/intel/ros2_intel_realsense
         launch_ros.actions.Node(
-            package='realsense_ros2_camera', node_executable='realsense_ros2_camera',
+            package='realsense2_camera', executable="realsense2_camera_node",
```

Comment:
Looks like this is failing CI because of the double quotes - needs to be single quotes like it was